padle/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .padle import PADle
+
+__red_end_user_data_statement__ = "No personal data is stored."
+
+
+def setup(bot):
```

Comment:
Do you think you could rewrite this to:

```py
async def setup(bot):
    bott.add_cog(PADle(bot)) if not __import__('asyncio').iscoroutinefunction(bot.add_cog) else await bot.add_cog(PADle(bot))
```

bc of silly dev branch reasons.  in the future, bot.add_cog will be a coroutine, so rn to cross over, we're allowing it to be loaded both ways